core-api/src/main/java/org/silverpeas/core/persistence/datasource/model/jpa/JpaEntityReflection.java

```diff
@@ -37,69 +35,22 @@
  */
 public class JpaEntityReflection {
 
-  private static Method createDateMethod = null;
-  private static Method lastUpdateDateMethod = null;
-  private static Field createdBySetManuallyField = null;
-  private static Field lastUpdatedBySetManuallyField = null;
+  private static Field creatorSetManuallyField = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "creatorSetManuallyField" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1068)